.travis.yml

```diff
@@ -1,13 +1,4 @@
 language: node_js
 node_js:
-  - "6"
   - "8"
   - "10"
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - g++-4.9
-before_install:
-  - export CXX="g++-4.9" CC="gcc-4.9"
```

Comment:
Not sure if all of this is still needed, looking at the deps/devDeps I guess not.

Reply:
That's for JSON-LD, which is part of the formats-common package. I will have a look.

Reply:
Travis is happy, no need to spend more time on this :)